app/assets/scss/_edit_section.scss

```diff
@@ -0,0 +1,5 @@
+.question-hint {
```

Comment:
This is available in the digital marketplace frontend toolkit: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/blob/master/toolkit/scss/forms/_hint.scss